packages/ui/primitives/document-flow/add-subject.tsx

```diff
@@ -117,6 +140,54 @@ export const AddSubjectFormPartial = ({
                 </li>
               </ul>
             </div>
+
+            <Accordion type="multiple" className="mt-8">
+              <AccordionItem value="advanced-options no-underline">
+                <AccordionTrigger className="text-md text-left no-underline">
+                  Advanced Options
+                </AccordionTrigger>
+
+                <AccordionContent className="text-muted-foreground max-w-prose text-sm leading-relaxed">
+                  {hasDateField && (
+                    <div className="mt-2 flex flex-col">
+                      <Label htmlFor="date-format">
+                        Date Format <span className="text-muted-foreground">(Optional)</span>
+                      </Label>
+
+                      <Select
+                        defaultValue={getValues('email.dateFormat')}
+                        onValueChange={(value) => setValue('email.dateFormat', value)}
```

Comment:
Use the `<Controller>` component for this instead please 😄

Reply:
Oki Doki